benchmark/src/jmh/java/io/opentelemetry/benchmark/classes/HttpClass.java

```diff
@@ -0,0 +1,88 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.opentelemetry.benchmark.classes;
+
+import com.google.api.client.http.GenericUrl;
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpRequestFactory;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.servlet.ServletContextHandler;
+
+public class HttpClass {
+  private String contextPath = "/path";
+  private Integer port = 18888;
+
+  public Server buildJettyServer() {
+    System.setProperty("org.eclipse.jetty.util.log.class", "org.eclipse.jetty.util.log.StdErrLog");
+    System.setProperty("org.eclipse.jetty.LEVEL", "WARN");
+
+    Server jettyServer = new Server(new InetSocketAddress("localhost", port));
+    ServletContextHandler servletContext = new ServletContextHandler();
+
+    servletContext.addServlet(HttpClassServlet.class, contextPath);
+    jettyServer.setHandler(servletContext);
+    return jettyServer;
+  }
+
+  @WebServlet
+  public static class HttpClassServlet extends HttpServlet {
+    @Override
+    public void init(final ServletConfig config) throws ServletException {
+      super.init(config);
+    }
+
+    @Override
+    protected void doGet(final HttpServletRequest req, final HttpServletResponse resp)
+        throws ServletException, IOException {
+      try {
+        Thread.sleep(10);
+      } catch (Exception e) {
+      }
+      resp.setContentType("application/json");
+      resp.setStatus(HttpServletResponse.SC_OK);
+      resp.getWriter().println("{ \"status\": \"ok\"}");
+    }
+
+    @Override
+    protected void doPost(final HttpServletRequest req, final HttpServletResponse resp)
+        throws ServletException, IOException {
+      doGet(req, resp);
+    }
+  }
+
+  private HttpRequestFactory requestFactory = new NetHttpTransport().createRequestFactory();
+
+  public void executeRequest() {
+    String url = "http://localhost:" + port + contextPath;
+
+    try {
+      HttpRequest request = requestFactory.buildGetRequest(new GenericUrl(url));
+      request.setThrowExceptionOnExecuteError(false);
+      request.execute();
+    } catch (Exception ex) {
+      ex.printStackTrace();
```

Comment:
Should we bubble up the exception instead of catch/log? Would it fail the test? (is that good?)

Reply:
I think it's going to be cleaner if we go the bubble up path